Amazon.IonDotnet/Internals/Tree/SystemTreeReader.cs

```diff
@@ -128,11 +128,40 @@ public int GetLobByteSize()
 
         public virtual ISymbolTable GetSymbolTable() => _systemSymbols;
 
-        public IEnumerable<SymbolToken> GetTypeAnnotations()
+        public string[] GetTypeAnnotations()
+        {
+            List<string> annotations = new List<string>();
+            foreach (SymbolToken symbolToken in _current.GetTypeAnnotations())
+            {
+                annotations.Add(symbolToken.Text);
```

Comment:
If it's possible for `symbolToken.Text` to be `null` here, we'll need to throw `UnknownSymbolException` in that case.

Reply:
Fixed.